Darktrace/manifest.json

```
@@ -5,7 +5,7 @@
     "type": "object",
     "properties": {
       "api_url": {
-        "description": "The url of the Darktrace appliance",
+        "description": "The url of the Darktrace appliance. e.g. https://example.darktrace.com or https://192.168.0.1",
```

JSON-schema allow you specifying the [format of the string](https://json-schema.org/understanding-json-schema/reference/string):
```suggestion
        "description": "The url of the Darktrace appliance. (e.g. https://example.darktrace.com or https://192.168.0.1)",
        "format": "uri"
```